documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -386,30 +403,70 @@ HEAD /v2/cluster
 
 If the preceding request returns a successful response code then the {brandname} REST server is running and serving requests.
 
-[[rest_server_configuration]]
-=== Retrieving Cache Configuration
-To retrieve a {brandname} configuration, invoke a `GET` request.
+[[rest_server_counters]]
+== Counter
+Use the REST API to work with counters.
+
+[[rest_server_counters_create]]
+=== Creating a Counter
+
+To create a counter, use a `POST` request with the configuration as payload.
 
 [source,options="nowrap",subs=attributes+]
 ----
-GET /v2/configurations/{name}
+POST /v2/counters/{counterName}
 ----
 
-.Headers
-|===
-|Header |Required or Optional |Parameter
+The payload is must contain a JSON configuration of the counter. Examples:
 
-|`Accept`
-|OPTIONAL
-|Sets the required format to return content. Supported formats are `application/xml` and `application/json`. The default is `application/json`. See link:#rest_accept[Accept] for more information.
-|===
+[source,json]
+----
+{
+    "weak-counter":{
+        "initial-value":5,
+        "storage":"PERSISTENT",
+        "concurrency-level":1
+    }
+}
+----
 
-[[rest_server_counters]]
-=== Counter
-Use the REST API to work with counters.
+[source,json]
+----
+{
+    "strong-counter":{
+        "initial-value":3,
+        "storage":"PERSISTENT",
+        "upper-bound":{
```

Comment:
can be changed to `"upper-bound": <value>`?

Reply:
It currently follows the XML structure (XML elements becomes json elements amd XML attr. becomes json fields). The idea is to make it easier for the user to go from the two.